web_frontend/osmaxx/job_progress/views.py

```diff
@@ -16,33 +17,49 @@ def tracker(request, order_id):
 
     emissary = Emissary(recipient=order.orderer)
 
+    substitutions = dict(
+        order_id=order.id,
+        excerpt_name=order.excerpt_name,
+        order_state=ExtractionOrderState.label(order.state),
+        order_url=request.build_absolute_uri(order.get_absolute_url()),
+    )
+
     if order.are_downloads_ready:
-        message = _('The extraction of the order "{order_id}" has been finished.').format(order_id=order.id)
+        message = _(
+            'The extraction of the order #{order_id} "{excerpt_name}" has been finished.'
+        ).format(**substitutions)
 
-        finished_email_subject = _('Extraction Order "{order_id} finished'.format(order_id=order.id))
+        finished_email_subject = _('Extraction Order #{order_id} "{excerpt_name}" finished').format(**substitutions)
```

Comment:
Consider using templates and rendering them instead of string interpolation. Is valid for almost all string interpolations we are doing, not commenting this below.